src/content/cnipluginmatrix/support_matrix.md

```diff
@@ -0,0 +1,24 @@
+## Support Matrix
+
+Submariner is designed to be cloud provider agnostic, and should run in any standard Kubernetes cluster. Submariner has been tested with and
+known to be working properly with the following cloud environments:
+
+> **_Note:_** Are you using Submariner in an environment which is not described here? Please [let us know](../contributing/website) so that we can update this document.
+
+### Cloud Providers
+
+* AWS
+* VMware vSphere
+* OpenStack
+* GCP
+
+### Kubernetes CNI Plugins
+
+Presently, Submariner has been tested with the following CNI Plugins that leverage kube-proxy with iptables mode:
+
+* [openshift-sdn](https://github.com/openshift/sdn)
```

Comment:
```suggestion
* [OpenShift-SDN](https://github.com/openshift/sdn)
```

I know it's just being moved, but a good opportunity to update it.

Reply:
Done